### ISSUE TITLE
Date field needs more flexibility

### ISSUE THREAD

**Issue by transat** [MukurtuCMS/mukurtucms#11]
The date field does not seem to accept more than a certain amount of characters, making date ranges such as `1840 - ca. 1855`, `circa 1900` or `Dec 1837 - Feb 1838` impossible to input.

**Comment by steve-taylor-wsu** [comment]
Correct, it only wants dates in the format of "YYYY", "YYYY-MM", or "YYYY-MM-DD".  I'll send your examples around and see how others are handling them.  Thanks

**Comment by transat** [comment]
Thanks. The current workaround is to add the more precise dates in the description field.
